stately-common/src/nativeCommonMain/kotlin/co/touchlab/stately/annotation/Annotations.kt

```diff
@@ -17,6 +17,4 @@
 
 package co.touchlab.stately.annotation
 
-import kotlin.native.Throws
-
-actual typealias Throws = Throws
+actual typealias Throws = kotlin.Throws
```

Comment:
`kotlin.Throws` is in common, so we could actually go further and deprecate the expect declaration for `co.touchlab.stately.annotation.Throws`. But that doesn't need to happen now if we just want to get a release out.

Reply:
Hmm. Deprecating is pretty easy. Might as well.